src/lib/services/email-service.ts

```diff
@@ -31,6 +31,8 @@ export interface IEmailEnvelope {
 
 const RESET_MAIL_SUBJECT = 'Unleash - Reset your password';
 const GETTING_STARTED_SUBJECT = 'Welcome to Unleash';
+const SCHEDULED_EXECUTION_FAILED_SUBJECT =
+    'Unleash - Scheduled execution failed';
```

Comment:
I'd make the subject line a little clearer. Scheduled execution of what? How about this?

```suggestion
    'Unleash - Scheduled change request could not be applied';
```